cmd/operator-sdk/internal/genutil/k8s.go

```diff
@@ -64,9 +63,6 @@ func deepcopyGen(hf string, fqApis []string) error {
 	if err != nil {
 		return err
 	}
-	if err := flag.Set("logtostderr", "true"); err != nil {
-		return err
-	}
```

Comment:
See: https://github.com/operator-framework/operator-sdk/pull/2702#discussion_r399829755

Reply:
It looks like its still being set [here](https://github.com/kubernetes/code-generator/blob/release-1.17/cmd/deepcopy-gen/main.go#L68) but the error isn't being checked so it could be erroring.

Reply:
Regardless this flag [defaults to true](https://github.com/kubernetes/klog/blob/v1.0.0/klog.go#L406), so we can remove this line.